<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml"><head><title>R: Quantitative proteomics data imputation</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="R.css" />
</head><body>

<table width="100%" summary="page for impute-methods {MSnbase}"><tr><td>impute-methods {MSnbase}</td><td style="text-align: right;">R Documentation</td></tr></table>

<h2> Quantitative proteomics data imputation </h2>

<h3>Description</h3>

<p> The <code>impute</code> method performs data imputation on an
<code>MSnSet</code> instance using a variety of methods (see below).  The
imputation and the parameters are logged into the
<code>processingData(object)</code> slot.
</p>
<p>Users should proceed with care when imputing data and take precautions
to assure that the imputation produce valid results, in particular
with naive imputations such as replacing missing values with 0.
</p>


<h3>Details</h3>

<p>There are two types of mechanisms resulting in missing values in
LC/MSMS experiments.
</p>

<ul>
<li><p>Missing values resulting from absence of detection of a
feature, despite ions being present at detectable concentrations.
For example in the case of ion suppression or as a result from the
stochastic, data-dependent nature of the MS acquisition
method. These missing value are expected to be randomly
distributed in the data and are defined as missing at random (MAR)
or missing completely at random (MCAR). 
</p>
</li>
<li><p>Biologically relevant missing values resulting from the
absence of the low abundance of ions (below the limit of detection
of the instrument). These missing values are not expected to be
randomly distributed in the data and are defined as missing not at
random (MNAR). 
</p>
</li></ul>

<p>MNAR features should ideally be imputed with a left-censor method,
such as <code>QRILC</code> below. Conversely, it is recommended to use host
deck methods such nearest neighbours, Bayesian missing value
imputation or maximum likelihood methods when values are missing at
random.
</p>
<p>Currently, the following imputation methods are available:
</p>

<dl>
<dt>MLE</dt><dd><p>Maximum likelihood-based imputation method using the EM
algorithm. Implemented in the <code>norm::imp.norm</code> function. See
<code><a href="../../norm/html/imp.norm.html">imp.norm</a></code> for details and additional
parameters. Note that here, <code>...</code> are passed to the
<code><a href="../../norm/html/em.norm.html">em.norm</a></code> function, rather to the actual
imputation function <code>imp.norm</code>. </p>
</dd>
<dt>bpca</dt><dd><p>Bayesian missing value imputation are available, as
implemented in the and <code>pcaMethods::pca</code> functions. See
<code><a href="../../pcaMethods/html/pca.html">pca</a></code> for details and additional
parameters. </p>
</dd>
<dt>knn</dt><dd><p>Nearest neighbour averaging, as implemented in the
<code>impute::impute.knn</code> function. See
<code><a href="../../impute/html/impute.knn.html">impute.knn</a></code> for details and additional
parameters. </p>
</dd>
<dt>QRILC</dt><dd><p>A missing data imputation method that performs the
imputation of left-censored missing data using random draws from a
truncated distribution with parameters estimated using quantile
regression. Implemented in the <code>imputeLCMD::impute.QRILC</code>
function. See <code><a href="../../imputeLCMD/html/impute.QRILC.html">impute.QRILC</a></code> for details
and additional parameters. </p>
</dd>
<dt>MinDet</dt><dd><p>Performs the imputation of left-censored missing data
using a deterministic minimal value approach. Considering a
expression data with <em>n</em> samples and <em>p</em> features, for
each sample, the missing entries are replaced with a minimal value
observed in that sample. The minimal value observed is estimated as
being the q-th quantile (default <code>q = 0.01</code>) of the observed
values in that sample. Implemented in the
<code>imputeLCMD::impute.MinDet</code> function. See
<code><a href="../../imputeLCMD/html/impute.MinDet.html">impute.MinDet</a></code> for details and additional
parameters. </p>
</dd>
<dt>MinProb</dt><dd><p>Performs the imputation of left-censored missing data
by random draws from a Gaussian distribution centred to a minimal
value. Considering an expression data matrix with <em>n</em> samples
and <em>p</em> features, for each sample, the mean value of the
Gaussian distribution is set to a minimal observed value in that
sample. The minimal value observed is estimated as being the q-th
quantile (default <code>q = 0.01</code>) of the observed values in that
sample. The standard deviation is estimated as the median of the
feature standard deviations. Note that when estimating the
standard deviation of the Gaussian distribution, only the
peptides/proteins which present more than 50% recorded values are
considered. Implemented in the <code>imputeLCMD::impute.MinProb</code>
function. See <code><a href="../../imputeLCMD/html/impute.MinProb.html">impute.MinProb</a></code> for details
and additional parameters. </p>
</dd>
<dt>min</dt><dd><p>Replaces the missing values by the smallest non-missing
value in the data. </p>
</dd>
<dt>zero</dt><dd><p>Replaces the missing values by 0.</p>
</dd>
<dt>mixed</dt><dd><p>A mixed imputation applying two methods (to be defined
by the user as <code>mar</code> for values missing at random and
<code>mnar</code> for values missing not at random, see example) on two
M[C]AR/MNAR subsets of the data (as defined by the user by a
<code>randna</code> logical, of length equal to <code>nrow(object)</code>). </p>
</dd>
<dt>nbavg</dt><dd><p>Average neighbour imputation for fractions collected
along a fractionation/separation gradient, such as sub-cellular
fractions. The method assumes that the fraction are ordered along
the gradient and is invalid otherwise.
</p>
<p>Continuous sets <code>NA</code> value at the beginning and the end of
the quantitation vectors are set to the lowest observed value in
the data or to a user defined value passed as argument <code>k</code>.
Them, when a missing value is flanked by two non-missing
neighbouring values, it is imputed by the mean of its direct
neighbours. A stretch of 2 or more missing values will not be
imputed. See the example below.
</p>
</dd>
</dl>

<p>The <code>naset</code> <code><a href="MSnSet-class.html">MSnSet</a></code> is an real quantitative
data where quantitative values have been replaced by <code>NA</code>s. See
<code>script/naset.R</code> for details.
</p>


<h3>Methods</h3>


<dl>
<dt><code>signature(object = "MSnSet", method, ...)</code></dt><dd><p> This method
performs data imputation on the <code>object</code> <code>MSnSet</code>
instance using the <code>method</code> algorithm. <code>...</code> is used to
pass parameters to the imputation function. See the respective
methods for details and additional parameters.</p>
</dd>
</dl>



<h3>Author(s)</h3>

<p>Laurent Gatto and Samuel Wieczorek
</p>


<h3>References</h3>

<p>Olga Troyanskaya, Michael Cantor, Gavin Sherlock, Pat Brown, Trevor
Hastie, Robert Tibshirani, David Botstein and Russ B.  Altman, Missing
value estimation methods for DNA microarrays Bioinformatics (2001) 17
(6): 520-525.
</p>
<p>Oba et al., A Bayesian missing value estimation method for gene
expression profile data, Bioinformatics (2003) 19 (16): 2088-2096.
</p>
<p>Cosmin Lazar (2015). imputeLCMD: A collection of methods for
left-censored missing data imputation. R package version
2.0. <a href="http://CRAN.R-project.org/package=imputeLCMD">http://CRAN.R-project.org/package=imputeLCMD</a>.
</p>
<p>Lazar C, Gatto L, Ferro M, Bruley C, Burger T. Accounting for the
Multiple Natures of Missing Values in Label-Free Quantitative Proteomics
Data Sets to Compare Imputation Strategies. J Proteome Res. 2016 Apr
1;15(4):1116-25. doi: 10.1021/acs.jproteome.5b00981. PubMed PMID:
26906401.
</p>


<h3>Examples</h3>

<pre>
data(naset)
## table of missing values along the rows
table(fData(naset)$nNA)
## table of missing values along the columns
pData(naset)$nNA

## non-random missing values
notna &lt;- which(!fData(naset)$randna)
length(notna)
notna

impute(naset, method = "min")

if (require("imputeLCMD")) {
    impute(naset, method = "QRILC")
    impute(naset, method = "MinDet")
}

if (require("norm"))
    impute(naset, method = "MLE")

impute(naset, "mixed",
       randna = fData(naset)$randna,
       mar = "knn", mnar = "QRILC")

## neighbour averaging

x &lt;- naset[1:4, 1:6]
exprs(x)[1, 1] &lt;- NA ## min value
exprs(x)[2, 3] &lt;- NA ## average
exprs(x)[3, 1:2] &lt;- NA ## min value and average
## 4th row: no imputation
exprs(x)

exprs(impute(x, "nbavg"))
</pre>

<hr /><div style="text-align: center;">[Package <em>MSnbase</em> version 2.4.2 <a href="00Index.html">Index</a>]</div>
</body></html>
