<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head><title>R: Visualizing and Analyzing Mass Spectrometry Related Data in
Proteomics</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="R.css" />
</head><body>
<h1> Visualizing and Analyzing Mass Spectrometry Related Data in
Proteomics
<img class="toplogo" src="../../../doc/html/Rlogo.svg" alt="[R logo]" />
</h1>
<hr/>
<div style="text-align: center;">
<a href="../../../doc/html/packages.html"><img class="arrow" src="../../../doc/html/left.jpg" alt="[Up]" /></a>
<a href="../../../doc/html/index.html"><img class="arrow" src="../../../doc/html/up.jpg" alt="[Top]" /></a>
</div><h2>Documentation for package &lsquo;protViz&rsquo; version 0.4.0</h2>

<ul><li><a href="../DESCRIPTION">DESCRIPTION file</a>.</li>
<li><a href="../doc/index.html">User guides, package vignettes and other documentation.</a></li>
<li><a href="../demo">Code demos</a>.  Use <a href="../../utils/help/demo">demo()</a> to run them.</li>
<li><a href="../NEWS">Package NEWS</a>.</li>
</ul>

<h2>Help Pages</h2>


<table width="100%">
<tr><td style="width: 25%;"><a href="AA.html">AA</a></td>
<td>AA - amino acid table</td></tr>
<tr><td style="width: 25%;"><a href="aa2mass.html">aa2mass</a></td>
<td>determine the weight if a fiven amino acid sequence</td></tr>
<tr><td style="width: 25%;"><a href="fetuinLFQ.html">apex</a></td>
<td>fetuinLFQ - A data set for evaluation of relative and absolute label-free quantification methods.</td></tr>
<tr><td style="width: 25%;"><a href="fragmentIons.html">as.data.frame.fragmentIonSet</a></td>
<td>Compute the b and y Fragment Ions of a Peptide Sequence</td></tr>
<tr><td style="width: 25%;"><a href="mascot.html">as.data.frame.mascot</a></td>
<td>Generic methods for mascot results</td></tr>
<tr><td style="width: 25%;"><a href="psm.html">as.data.frame.psmSet</a></td>
<td>Compute a matching between a peptide sequence and a MS2 spectrum</td></tr>
<tr><td style="width: 25%;"><a href="mascot.html">as.psm.mascot_query</a></td>
<td>Generic methods for mascot results</td></tr>
<tr><td style="width: 25%;"><a href="psmSet.html">as.psmSet</a></td>
<td>psmSet - a set of peptide spectrum matches</td></tr>
<tr><td style="width: 25%;"><a href="psmSet.html">as.psmSet.mascot</a></td>
<td>psmSet - a set of peptide spectrum matches</td></tr>
<tr><td style="width: 25%;"><a href="averagine.html">averagine</a></td>
<td>averagine - a data set conatining isotope envelopes of averagine peptides</td></tr>
<tr><td style="width: 25%;"><a href="fragmentIons.html">bymatrix</a></td>
<td>Compute the b and y Fragment Ions of a Peptide Sequence</td></tr>
<tr><td style="width: 25%;"><a href="fragmentIons.html">defaultIon</a></td>
<td>Compute the b and y Fragment Ions of a Peptide Sequence</td></tr>
<tr><td style="width: 25%;"><a href="deisotoper.html">deisotoper</a></td>
<td>find isotop pattern in a given mass spectrum</td></tr>
<tr><td style="width: 25%;"><a href="de_novo.html">de_novo</a></td>
<td>de-novo on tandem ms</td></tr>
<tr><td style="width: 25%;"><a href="fetuinLFQ.html">empai</a></td>
<td>fetuinLFQ - A data set for evaluation of relative and absolute label-free quantification methods.</td></tr>
<tr><td style="width: 25%;"><a href="fasta.html">FASTA</a></td>
<td>FASTA format reader</td></tr>
<tr><td style="width: 25%;"><a href="fasta.html">Fasta</a></td>
<td>FASTA format reader</td></tr>
<tr><td style="width: 25%;"><a href="fasta.html">fasta</a></td>
<td>FASTA format reader</td></tr>
<tr><td style="width: 25%;"><a href="fetuinLFQ.html">fetuinLFQ</a></td>
<td>fetuinLFQ - A data set for evaluation of relative and absolute label-free quantification methods.</td></tr>
<tr><td style="width: 25%;"><a href="PTM_MarkerFinder.html">findMz</a></td>
<td>PTM MarkerFinder</td></tr>
<tr><td style="width: 25%;"><a href="PTM_MarkerFinder.html">findMz.mascot</a></td>
<td>PTM MarkerFinder</td></tr>
<tr><td style="width: 25%;"><a href="PTM_MarkerFinder.html">findMz.psmSet</a></td>
<td>PTM MarkerFinder</td></tr>
<tr><td style="width: 25%;"><a href="findNN.html">findNN</a></td>
<td>find index of nearest neighbor</td></tr>
<tr><td style="width: 25%;"><a href="findNN.html">findNN_</a></td>
<td>find index of nearest neighbor</td></tr>
<tr><td style="width: 25%;"><a href="fragmentIons.html">fragmentIon</a></td>
<td>Compute the b and y Fragment Ions of a Peptide Sequence</td></tr>
<tr><td style="width: 25%;"><a href="genMod.html">genMod</a></td>
<td>Generates all possible modification of a given peptide seqence and modification pattern</td></tr>
<tr><td style="width: 25%;"><a href="HexNAc.html">HexNAc</a></td>
<td>HexNAc - Analysis of N-HexNAc glycopeptides by LC-MS/MS, using HCD and ETD fragmentation techniques</td></tr>
<tr><td style="width: 25%;"><a href="ssrc.html">hydrophobicity</a></td>
<td>Sequence Specific Retention Calculator</td></tr>
<tr><td style="width: 25%;"><a href="fragmentIons.html">ionseries</a></td>
<td>Compute the b and y Fragment Ions of a Peptide Sequence</td></tr>
<tr><td style="width: 25%;"><a href="iRTpeptides.html">iRT</a></td>
<td>iRT peptides - independent retention time peptides</td></tr>
<tr><td style="width: 25%;"><a href="iRTpeptides.html">irt</a></td>
<td>iRT peptides - independent retention time peptides</td></tr>
<tr><td style="width: 25%;"><a href="iRTpeptides.html">iRTpeptides</a></td>
<td>iRT peptides - independent retention time peptides</td></tr>
<tr><td style="width: 25%;"><a href="mascot.html">is.mascot</a></td>
<td>Generic methods for mascot results</td></tr>
<tr><td style="width: 25%;"><a href="mascot.html">is.mascot_query</a></td>
<td>Generic methods for mascot results</td></tr>
<tr><td style="width: 25%;"><a href="psm.html">is.psm</a></td>
<td>Compute a matching between a peptide sequence and a MS2 spectrum</td></tr>
<tr><td style="width: 25%;"><a href="psmSet.html">is.psmSet</a></td>
<td>psmSet - a set of peptide spectrum matches</td></tr>
<tr><td style="width: 25%;"><a href="iTRAQ.html">iTRAQ</a></td>
<td>iTRAQ - A small 8-plex iTRAQ data set with confident identified peptides from 5 proteins.</td></tr>
<tr><td style="width: 25%;"><a href="iTRAQ2GroupAnalysis.html">iTRAQ2GroupAnalysis</a></td>
<td>iTRAQ two group analysis</td></tr>
<tr><td style="width: 25%;"><a href="lcmsmap.html">lcmsmap</a></td>
<td>LC-MS Map</td></tr>
<tr><td style="width: 25%;"><a href="lcmsmap.html">lcmsoverview</a></td>
<td>LC-MS Map</td></tr>
<tr><td style="width: 25%;"><a href="findNN.html">lower_bound_</a></td>
<td>find index of nearest neighbor</td></tr>
<tr><td style="width: 25%;"><a href="mascot.html">mascot</a></td>
<td>Generic methods for mascot results</td></tr>
<tr><td style="width: 25%;"><a href="mdp.html">massDeviationPlot</a></td>
<td>Mass Deviation Plot</td></tr>
<tr><td style="width: 25%;"><a href="mdp.html">mdp</a></td>
<td>Mass Deviation Plot</td></tr>
<tr><td style="width: 25%;"><a href="msms.html">msms</a></td>
<td>A data set containing tandem mass spectra of an LCMS experiment.</td></tr>
<tr><td style="width: 25%;"><a href="findNN.html">NN</a></td>
<td>find index of nearest neighbor</td></tr>
<tr><td style="width: 25%;"><a href="parentIonMass.html">parentIonMass</a></td>
<td>Compute Parent Ion Mass of a Peptide Sequence</td></tr>
<tr><td style="width: 25%;"><a href="peakplot.html">peakplot</a></td>
<td>Labelling of Peptide Fragment Mass Spectra</td></tr>
<tr><td style="width: 25%;"><a href="peakplot.html">peakplot.label</a></td>
<td>Labelling of Peptide Fragment Mass Spectra</td></tr>
<tr><td style="width: 25%;"><a href="peakplot.html">peakplot.pie</a></td>
<td>Labelling of Peptide Fragment Mass Spectra</td></tr>
<tr><td style="width: 25%;"><a href="peakplot.html">peakplot.putlabel</a></td>
<td>Labelling of Peptide Fragment Mass Spectra</td></tr>
<tr><td style="width: 25%;"><a href="lcmsmap.html">pep3d</a></td>
<td>LC-MS Map</td></tr>
<tr><td style="width: 25%;"><a href="parentIonMass.html">pepmass</a></td>
<td>Compute Parent Ion Mass of a Peptide Sequence</td></tr>
<tr><td style="width: 25%;"><a href="pgImporter.html">pgImporter</a></td>
<td>Progenesis Importer Function</td></tr>
<tr><td style="width: 25%;"><a href="pgLFQaov.html">pgLFQaov</a></td>
<td>iTRAQ two group analysis</td></tr>
<tr><td style="width: 25%;"><a href="pgLFQfeature.html">pgLFQfeature</a></td>
<td>pgLFQfeature - A data set with a featuremap export ProgenesisLCMS</td></tr>
<tr><td style="width: 25%;"><a href="pgLFQprot.html">pgLFQprot</a></td>
<td>pgLFQprot - A data set with a featuremap export ProgenesisLCMS</td></tr>
<tr><td style="width: 25%;"><a href="pgLFQtNpq.html">pgLFQtNpq</a></td>
<td>Label Free Quantification using the top N peptide approach</td></tr>
<tr><td style="width: 25%;"><a href="parentIonMass.html">pim</a></td>
<td>Compute Parent Ion Mass of a Peptide Sequence</td></tr>
<tr><td style="width: 25%;"><a href="mascot.html">plot.mascot</a></td>
<td>Generic methods for mascot results</td></tr>
<tr><td style="width: 25%;"><a href="mascot.html">plot.mascot_query</a></td>
<td>Generic methods for mascot results</td></tr>
<tr><td style="width: 25%;"><a href="peakplot.html">plot.psm</a></td>
<td>Labelling of Peptide Fragment Mass Spectra</td></tr>
<tr><td style="width: 25%;"><a href="lcmsmap.html">plot.psmSet</a></td>
<td>LC-MS Map</td></tr>
<tr><td style="width: 25%;"><a href="ppp.html">ppp</a></td>
<td>Plotting pressure profile data from Eksigent LC pumps</td></tr>
<tr><td style="width: 25%;"><a href="pps.html">pps</a></td>
<td>A misc function for finding NN time slots in pressure profile data.</td></tr>
<tr><td style="width: 25%;"><a href="PTM_MarkerFinder.html">pPTM</a></td>
<td>PTM MarkerFinder</td></tr>
<tr><td style="width: 25%;"><a href="pressureProfile.html">pressureProfile</a></td>
<td>NanoLC pressure profile</td></tr>
<tr><td style="width: 25%;"><a href="ppp.html">pressureProfilePlot</a></td>
<td>Plotting pressure profile data from Eksigent LC pumps</td></tr>
<tr><td style="width: 25%;"><a href="pps.html">pressureProfileSummary</a></td>
<td>A misc function for finding NN time slots in pressure profile data.</td></tr>
<tr><td style="width: 25%;"><a href="peakplot.html">protViz</a></td>
<td>Labelling of Peptide Fragment Mass Spectra</td></tr>
<tr><td style="width: 25%;"><a href="psm.html">psm</a></td>
<td>Compute a matching between a peptide sequence and a MS2 spectrum</td></tr>
<tr><td style="width: 25%;"><a href="PTM_MarkerFinder.html">PTM_MarkerFinder</a></td>
<td>PTM MarkerFinder</td></tr>
<tr><td style="width: 25%;"><a href="PTM_MarkerFinder_util.html">PTM_MarkerFinder_util</a></td>
<td>PTM MarkerFinder util plot</td></tr>
<tr><td style="width: 25%;"><a href="ssrc.html">ssrc</a></td>
<td>Sequence Specific Retention Calculator</td></tr>
<tr><td style="width: 25%;"><a href="mascot.html">summary.mascot</a></td>
<td>Generic methods for mascot results</td></tr>
<tr><td style="width: 25%;"><a href="psm.html">summary.psmSet</a></td>
<td>Compute a matching between a peptide sequence and a MS2 spectrum</td></tr>
<tr><td style="width: 25%;"><a href="fetuinLFQ.html">t3pq</a></td>
<td>fetuinLFQ - A data set for evaluation of relative and absolute label-free quantification methods.</td></tr>
<tr><td style="width: 25%;"><a href="pgLFQtNpq.html">tNpq</a></td>
<td>Label Free Quantification using the top N peptide approach</td></tr>
</table>
</body></html>
